doc/guide/introduction/tutorial.xml

```diff
@@ -315,4 +329,52 @@
           <video xml:id="gsv-2023-jan-video" youtube="y-33Ube-Rhk"/>
       </figure>
     </section>
+
+  <section xml:id="quickstart-example">
+    <title>For Advanced Users New to PreTeXt</title>
+
+    <p>
+      This section is intended for users who have read
+      <xref ref="start-here"/>
+      and are already experienced
+      with other markup languages and command-line tools. If you're
+      interested in a more turnkey workflow to getting started with
```

Comment:
"turnkey" isn't familiar to me, and may not be to others (especially non-English-first-language users).  Is there another term for that?

Reply:
https://www.merriam-webster.com/thesaurus/turnkey comes up with nothing slick - I'm a single-language English speaker but feel it's not too uncommon. (Specific suggestions from others are welcome.)